unobot.py

```diff
@@ -462,14 +463,14 @@ def get_card(self):
     @staticmethod
     def create_deck():
         new_deck = []
-        for card in COLORED_CARD_NUMS:
+        for card in (COLORED_CARD_NUMS + COLORED_CARD_NUMS[1:]):
```

Comment:
Solved by adding twice as many W/WD4 cards in c26a696. That adds 4 more cards for a total of **108**, which is the regulation deck size.